### ISSUE TITLE
How to generate a n-tissue segmentation with only one tissue (+  DTI derived maps) ?

### ISSUE THREAD

**Issue by valeryozenne** [ANTsX/ANTs#1146]
Dear ANTs experts,

I'm looking for a bit of help, I 'm mostly looking for guidance & advises. I think  ANTs has been designed to solve this kind of problem I've got but I'm not 100% sure as my images and the corresponding tissue properties slightly differs from neuroimaging studies. 

## Objective 

My objective is to generate a parcellization map in cardiac samples for each subject, then I wish to apply the MALF algorithm to create a kind of standardized parcellization map based on structural data. However, I'm stuck with the parcellization step. I've never done such a thing, so I wonder if you could indicate some guidelines. Please find below some precision

## Available data
- anatomical data  (in native and template space)
- diffusion data  (in native and template space)

## Limitation
- my anatomical data includes only 1 tissu  (there is no GM & WM contrast  ). In the exemple below, some slight changes in contrast but adjacent structures are visible but it is not always the case for other structures   
- thresholding on FA  does not allow to segment the structures I'm interesting in like for GM or WM.
- there is no available parallelization map 
- there is evidence (see images below ) that DTI derived maps (the eigenvectors in RGB representation) could be used to do the parcellization.
- I'm currently playing with antsAtroposN4.sh with my anatomical images in input, it is running and clustering the intensity but not in the wished way (but I'm really far from understanding with Atropos does and can do). 

## Questions

- Is it possible to feed antsAtroposN4.sh with eigenvectors or RGB maps or x, y, z components together  ? Is it relevant ?
- Does ANTs provide another tools that could help in solving my problem ? 
- If my strategy is not clear / make no sense, don't hesitate to tell me. I'll be happy to discover how to solve this question. 

Thanks in advance,
Valéry

## Figures

For instance , I wish to extract a parcellization map based on vector orientation that define the singularity in blue (that has a triangular shape) at the intersection of the junction.  

Data for each subject in template space:
![Diapositive6_for_ants](https://user-images.githubusercontent.com/7050315/109146139-b78ef880-7763-11eb-8fd6-77bffe32fb55.PNG)
Template images  in template space:
![Diapositive1_for_ants](https://user-images.githubusercontent.com/7050315/109146155-bc53ac80-7763-11eb-9a5e-5085ad0aeb29.PNG)

**Comment by ntustison** [comment]
I'm skeptical that any current ANTs tools would be able to give you what you wanted.  Both Atropos and joint label fusion handle multiple channels but I don't think either one is going to get you what you want.  For something like this, I would look at deep learning possibilities.  However, you'd have to build your own pipeline and create your own training data, although we do have some examples that would get you started in ANTsRNet/ANTsPyNet.

**Comment by valeryozenne** [comment]
Thanks for advises. Using ANtsPyNet is on my todo list! But unfortunately my learning phase is quite long in general and my comprehension of CNN is still poor. I found interesting for my projects the paper on functional lung MRI with template generation and data augmentation, I usually have between 2 to 100 samples than a thousand.  

I will give a try with Atropos in multiple channels as a starting point. Maybe some practical questions will follow. Best,
